content/projects/gsoc/2023/projects/add-probes-to-plugin-health-score.adoc

```diff
@@ -9,36 +9,53 @@ tags:
 - plugins
 status: "Selected"
 sig: platform
-//FIXME: create bio for Jagruti Tiwari in content/_data/authors/
-student: "null_gsoc_contributor"
+student: "jagruti"
 mentors:
 - "alecharp"
 - "dheerajodha"
 - "mostafaashraf"
 links:
     idea: /projects/gsoc/2023/project-ideas/add-probes-to-plugin-health-score
-//   gitter: "some gitter channel"
-//   draft: link to the draft/final proposal
-//   meeting: link to internal section of this page with the meeting details and notes
+//   gitter: https://app.element.io/#/room/#jenkinsci_GSoC-Plugin_Health_Score:gitter.im
+//   draft: https://docs.google.com/document/d/14URAapu7svkj1pgw3yoCybwn766Net9SZ9coC4zx7yg/edit
+//   meeting: https://docs.google.com/document/d/1QcwSiAuQtoy4dGlPXgY3w8FjDzTJn-3yCv75U-OFJ04/edit
 ---
-
-//above links must be updated
-
 === Abstract
 
-(to be described)
+The project proposes to add new probes to the Plugin Health Score tool. Originally started in 2022 as a part of a GSoC project, this tool determines the health score of a plugin. Probes are the heart of the tool. To extend the work started last year, I propose adding new probes to make the tool more valuable and feature-complete.
```

Comment:
As this page may evolve, I think it's easier to maintain if each sentence is on it's own line. From an AsciiDoc syntax point of view, it is not creating a "space" between the sentences once rendered.

(optional)

```suggestion
The project proposes to add new probes to the Plugin Health Score tool.
Originally started in 2022 as a part of a GSoC project, this tool determines the health score of a plugin.
Probes are the heart of the tool.
To extend the work started last year, I propose adding new probes to make the tool more valuable and feature-complete.
```